mtags/src/main/scala-3/scala/meta/internal/pc/ScalaPresentationCompiler.scala

```diff
@@ -247,7 +248,16 @@ case class ScalaPresentationCompiler(
                 case _: ImportType =>
                   printer.typeString(symbol.paramRef(using ctx))
                 case _ =>
-                  printer.typeString(tpw)
+                  val shortendType = driver.compilationUnits.get(uri) match {
+                    case Some(unit) =>
+                      implicit val newctx = ctx.fresh.setCompilationUnit(unit)
+                      val path = Interactive.pathTo(newctx.compilationUnit.tpdTree, pos.span)
+                      val context = Interactive.contextOfPath(path)
```

Comment:
Doesn't seem to work with that.

Reply:
I'm not 100% sure, but I guess it doesn't work because the `path` defined in line 226, is based on an untyped tree whose denotation doesn't have a corresponding symbol yet.
Therefore, here we recalculate the path using a typed tree (referring https://github.com/tanishiking/dotty/blob/64aa7cfb0a0a245261c490e268052880a07f5c00/compiler/src/dotty/tools/dotc/interactive/Completion.scala#L47)